### PR TITLE
Add warning when collective already has virtual cards assigned

### PR DESCRIPTION
**Pull request by hdiniz**
Resolve https://github.com/opencollective/opencollective/issues/6834
Require https://github.com/opencollective/opencollective-frontend/pull/8991

# Description

Display warning when the collective already has virtual and might be missing receipts.

# Screenshots

image

image

### CODE REVIEW

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |

File: components/edit-collective/EditVirtualCardModal.tsx [hunk 1]
```diff
@@ -102,6 +104,24 @@ const collectiveMembersQuery = gql`
...
+        hasMissingReceipts: true
+      ) {
+        totalCount
+      }
+    }
+  }
+`;
+
 const VirtualCardPoliciesQuery = gql`
   query VirtualCardPoliciesQuery($slug: String) {
```
Comment (znarf) [inline components/edit-collective/EditVirtualCardModal.tsx:126]: It's not new but the convention is that it should be `VirtualCardPolicies` and not `VirtualCardPoliciesQuery`

https://github.com/opencollective/opencollective/blob/main/rfcs/011-graphql.md

**Review by znarf — APPROVED** [review]
Looks good!

**Pull request merged**